### PR TITLE
Added iv calculator

### PR DESCRIPTION
**Pull request by Jari27**
**Changes made:** Added an IV Ratio calculator to the Pokemon class. Use like this

`double ivRatio = pokemon.getIVRatio()`

**Reason for change**: seemed useful.

### CODE REVIEW

**Comment by FabianTerhorst** [comment]
Could be helpful: http://www.pokewiki.de/Determinant_Values

File: src/main/java/com/pokegoapi/api/pokemon/Pokemon.java [hunk 1]
```diff
@@ -267,6 +267,14 @@ public int getIndividualDefense() {
 	public int getIndividualStamina() {
 		return proto.getIndividualStamina();
 	}
+	
+	/**
+	 * Calculates the pokemons IV ratio
```
Comment (jabbink) [inline src/main/java/com/pokegoapi/api/pokemon/Pokemon.java]: Doesn't this break `checkstyleMain` because it does not contain a dot?

**Comment by FabianTerhorst** [comment]
http://bulbapedia.bulbagarden.net/wiki/Individual_values

File: src/main/java/com/pokegoapi/api/pokemon/Pokemon.java (hunk 1 above)
Comment (Jari27) [inline src/main/java/com/pokegoapi/api/pokemon/Pokemon.java]: I can't get checkstyle to run. If I need to change it I will. I can also start working on annotating more of the classes.

Comment (FabianTerhorst) [inline src/main/java/com/pokegoapi/api/pokemon/Pokemon.java]: just change `Calculates the pokemons IV ratio` to `Calculates the pokemons IV ratio.`

**Comment by FabianTerhorst** [comment]
only at the title, not at the return description.

**Comment by FabianTerhorst** [comment]
👍

**Pull request merged**